apps/docs/src/routes/concepts/miscellaneous.md

```diff
@@ -15,12 +15,12 @@ Use the property `lookAt` on an Object to
   let mesh
 </script>
 
-<PerspectiveCamera lookAt={mesh} />
-<Mesh bind:mesh … />
+<T.PerspectiveCamera lookAt={mesh} />
+<T.Mesh bind:ref={mesh} … />
 ```
 
 - to reactively orient an Object towards a Position
 
 ```svelte
-<PerspectiveCamera lookAt={{ x: 5, y: 3 }} />
+<T.PerspectiveCamera lookAt.x={5} lookAt.y={3} />
 ```
```

Comment:
`lookAt` actually doesn't work anymore, we should remove it.

Reply:
Should this whole file/route be removed then? Or leave the `## lookAt` section commented out for now?

Reply:
I guess we can remove the while route/file 👍 
Supporting `lookAt` will be possible using a plugin.